src/translations/en.json

```diff
@@ -69,18 +71,18 @@
     "CBAll": "All",
     "CBNone": "None",
     "Map": "Map",
-    "lineFullPath": "Line full path",
-    "lineParallelPath": "Spread overlapping lines",
-    "LineFlowMode": "Line flow mode",
+    "lineFullPath": "Line full path: ",
+    "lineParallelPath": "Spread overlapping lines: ",
+    "LineFlowMode": "Line flow mode: ",
     "None": "None",
     "StaticArrows": "Static arrows",
     "AnimatedArrows": "Animated arrows",
     "Feeders": "Feeder",
-    "LineFlowColorMode": "Line color mode",
+    "LineFlowColorMode": "Line color mode: ",
     "NominalVoltage": "Nominal voltage",
     "Overloads": "Overloads",
     "Overload": "Overload",
-    "AlertThresholdLabel": "Alert threshold",
+    "AlertThresholdLabel": "Threshold for displaying lines\n before overload (%): ",
```

Comment:
It seems that the newline is not taken into account ?

Reply:
Yes, it was added for test purpose and I forgot to remove it. Removed.